.gitignore

```diff
@@ -1,6 +1,7 @@
 .aws/
 .coverage
 .gitconfig
+.vscode/
```

Comment:
What's this from?  Is it part of the JIRA ticket?

Reply:
It’s not part of this ticket. VSCode sometimes makes this directory, and I had accidentally committed it at one point while working on this branch before realizing .vscode/ had been created.

Reply:
Are you removing it?  Or do you think we should keep it in the repo? 

I'm back and forth on it.  It might be beneficial to .gitignore standard personal development environment files, like .vscode, or in my case for vim, .swp files.  On the other hand an individual's development environment isn't really a part of the encoded project.  And it could get out of hand pretty quick.  There are alternate solutions too like adding it to the .gitignore_global in the home directory.

Reply:
I don’t really care. Some other people here use VSCode and might accidentally commit .vscode/ too, so I felt like this was a way to head off what I had done accidentally. If you prefer I used .gitignore_global, I’ll do that.

Reply:
Yea okay, let's leave it in but in general push people to using .gitignore_global for "personal" type things.

Reply:
That sounds good. Probably should document that somewhere.